commercial/app/controllers/commercial/package.scala

```diff
@@ -20,7 +20,7 @@ package object commercial {
   }
 
   def specificId(implicit request: RequestHeader): Option[String] = request.queryString.get("t").map(_.head)
-  def specificIds(implicit request: RequestHeader): Seq[String] = request.queryString.getOrElse("t", Nil).reverse
+  def specificIds(implicit request: RequestHeader): Seq[String] = request.queryString.getOrElse("t", Nil)
```

Comment:
I believe so; the values associated with `t` are added to the request in the order they are specified in DFP. Playing around with various IDs seemed to confirm it.